src/System.IO.FileSystem.Primitives/tests/System.IO.FileSystem.Primitives.Tests.csproj

```diff
@@ -7,7 +7,7 @@
     <ProjectGuid>{2EF7EFA5-F171-4CAB-8A29-32833949FD87}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.IO.FileSystem.Primitives.Tests</AssemblyName>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Once you add the netstandard1.3 build configuration you will need to conditions this on an empty targetgroup.